src/extension/windows/wpfPlatform.ts

```diff
@@ -30,7 +30,8 @@ export class WpfPlatform extends WindowsPlatform {
             },
         };
 
-        extProps = TelemetryHelper.addReactNativeVersionToEventProperties(this.runOptions.reactNativeVersion, extProps);
+        extProps = TelemetryHelper.addReactNativeVersionToEventProperties(this.runOptions.reactNativeVersions.reactNativeVersion, extProps);
```

Comment:
It seems that we get the version twice here.

Reply:
I didn't find getting of React Native version in that file.